src/Projections/Reunion/Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.cs

```diff
@@ -17,19 +20,57 @@ public DispatcherQueueSynchronizationContext(DispatcherQueue dispatcherQueue)
             m_dispatcherQueue = dispatcherQueue;
         }
 
-        public override void Post(SendOrPostCallback d, object state)
+        /// <inheritdoc/>
+        public override unsafe void Post(SendOrPostCallback d, object? state)
         {
-            if (d == null)
+#if NET6_0
```

Comment:
For simplicity, could this whole if/else/endif block be

if (d is null)
{
    ThrowArgumentNullException(nameof(d));
}

Looks like we get the same code but much easier to read.

Reply:
Done in 91f5a0881857d8e262e47fd6c47aa8d9cc6f73f9 🙂